src/PublicApiAnalyzers/UnitTests/DeclarePublicAPIAnalyzerTestsBase.cs

```diff
@@ -2344,8 +2352,8 @@ public async Task TypeArgumentRefersToTypeParameter_OnType_ObliviousReferenceAsy
             var shippedText = "#nullable enable";
             var unshippedText = @"";
             await VerifyCSharpAsync(source, shippedText, unshippedText,
-                // /0/Test0.cs(3,21): warning RS0016: Symbol 'C<T>' is not part of the declared API.
-                GetCSharpResultAt(3, 15 + EnabledModifierCSharp.Length, DeclareNewApiRule, "C<T>"),
+                // /0/Test0.cs(3,21): warning RS0016: Symbol '~C<T>' is not part of the declared API.
+                GetCSharpResultAt(3, 15 + EnabledModifierCSharp.Length, DeclareNewApiRule, "~C<T>"),
```

Comment:
Do we have a test with an API that now shows a nullable annotation (`?`)? For some reason, I didn't notice any in the updated tests, but I don't know why.